Automattic/wp-calypso
# Add OAuth Client name to the login title

This pull request updates the header text on the login page when a client id is set, to display the name of the OAuth2 client application.
  
#### Testing instructions
  
1. Run `git checkout add/oauth-title` and start your server
2. Open the `Login` page with a client id - http://calypso.localhost:3000/log-in?clientId=1
3. Check that you see the name of the client application in the header.
4. Try a different client id and check that you still see the same name 

#### Reviews
